plugins/postcss-design-tokens/package.json

```diff
@@ -45,7 +45,7 @@
 	},
 	"devDependencies": {
 		"postcss-import": "^14.1.0",
-		"style-dictionary-design-tokens-example": "^1.0.0"
+		"style-dictionary-design-tokens-example": "^1.1.0"
```

Comment:
The `"exports"` field was incorrect there.
Noticed it while trying to node module resolutions.

No work was done to resolve modules in this PR, but updating the package anyway.